### ISSUE TITLE
Consider collaborating with the d8vk project? 

### ISSUE THREAD

**Issue by LethalManBoob** [crosire/d3d8to9#148]
https://github.com/AlpyneDreams/d8vk
I think it would be great considering how complete d3d8 to d3d9 is already and it already works great with dxvk

**Comment by lorn10** [comment]
Yeah, and **d3d8to9** also works great with [Gallium Nine](https://github.com/iXit/wine-nine-standalone).

I think it would be therefore great if d3d8to9 could be added at some point to [winetricks](https://github.com/Winetricks/winetricks). @austin987 :wink: 

That would make the handling easier and the end-user has the convenient choice to use it with DXVK or Gallium Nine. :+1:

**Comment by WinterSnowfall** [comment]
@LethalManBoob Probably should have replied earlier here, but your wish did ultimately come true in away.

I've touched on [the differences between d8vk and d3d8to9 before](https://github.com/AlpyneDreams/d8vk/issues/114), but the TLDR version of it is d3d8to9's much more focused scope and its reliance on native d3d9, which is a boon in itself (while d8vk/dxvk essentially has to implement d3d8/9 on top of Vulkan, and nothing comes for free... the margin for error is also much greater).

That being said, as someone who has worked helped with d8vk and is still helping maintain the d3d8 code in dxvk, I've done my best to port over fixes for d3d8 games where that made sense. And I'll continue to do so... so this "issue" can be closed, rest asured :smile:.

**Comment by LethalManBoob** [comment]
> @LethalManBoob Probably should have replied earlier here, but your wish did ultimately come true in a way.
> 
> I've touched on [the differences between d8vk and d3d8to9 before](https://github.com/AlpyneDreams/d8vk/issues/114), but the TLDR version of it is d3d8to9's much more focused scope and its reliance on native d3d9, which is a boon in itself (while d8vk/dxvk essentially has to implement d3d8/9 on top of Vulkan, and nothing comes for free... the margin for error is also much greater).
> 
> That being said, as someone who has helped with d8vk and is still helping maintain the d3d8 code in dxvk, I've done my best to port over fixes for d3d8 games where that made sense. And I'll continue to do so... so this "issue" can be closed, rest asured 😄.

Aha. I had forgotten about this. Nice info. Thanks. Yeah this thread is done now. :D
